README.md

```diff
@@ -87,6 +87,44 @@ public void onBrowserSwitchResult(int requestCode, BrowserSwitchResult result, @
 }
 ```
 
+## BrowserSwitchClient
```

Comment:
This header is at the same level as `##Usage`. Should we make these two integration patterns their own unique headers? Also - I assume we prefer merchants to utilize the new pattern, so maybe we put it first.

Reply:
Yeah this is a good question. It's kind of an alternate usage right now. The original integration is probably the most seamless at the moment, but we need more control over how BrowserSwitchListener gets injected right now.

I think in v1.1.0, we may be able to leverage Activity Lifecycles to get a similarly seamless interface without having to have access to BrowserSwitchFragment. For now maybe `Alternative Usage: BrowserSwitchClient` would be a good title?

Reply:
Sounds good